src/main/java/org/saltyrtc/client/helpers/RandomHelper.java

```diff
@@ -13,14 +13,17 @@
 
 public class RandomHelper {
 
+    private static Random random = new Random();
+    private static SecureRandom secureRandom = new SecureRandom();
```

Comment:
To explain my change:

- When we use static members, we do not have to create new `Random`/`SecureRandom` objects for every method call.
- Although `Random` is not used for security-critical stuff, it's usually bad practice to create a new object every time you need a random value. At object creation it is seeded with the current time, so it could be possible to get the same random value when the method is called at the same time. For `SecureRandom` it should not matter because the CSPRNG of the OS is used.